portfolio/src/main/webapp/script.js

```diff
@@ -16,7 +16,20 @@
  * Adds a random greeting to the page.
  */
 
-$(document).ready(initClasses);
+$(document).ready(init);
+
+function init(){
+  initCardImgs();
+  initClasses();
+}
+
+function initCardImgs(){
+  $(".card-img").each( function(){
+    src = 'url("images/' + $(this).attr('id') +'.jpg")';
```

Comment:
I recommend using absolute paths here instead. It makes it much easier to move files around or support different url schemes. I.e. this would become `/images/`